ui/helpers/constants/error-keys.js

```diff
@@ -7,3 +7,4 @@ export const GAS_PRICE_EXCESSIVE_ERROR_KEY = 'gasPriceExcessive';
 export const UNSENDABLE_ASSET_ERROR_KEY = 'unsendableAsset';
 export const INSUFFICIENT_FUNDS_FOR_GAS_ERROR_KEY = 'insufficientFundsForGas';
 export const INVALID_ASSET_TYPE = 'invalidAssetType';
+export const IS_SIGNING_OR_SUBMITTING = 'isSigningOrSubmitting';
```

Comment:
How about using a clearer name for the constant to provide better context?

Reply:
I'd argue this has sufficient context since all these errors are in the context of transactions so signing and submitting are standard phases.